build.gradle

```diff
@@ -44,36 +44,37 @@ dependencies {
     compile project(':owncloud-android-library')
     compile "com.android.support:support-v4:${supportLibraryVersion}"
     compile "com.android.support:design:${supportLibraryVersion}"
-    compile 'com.jakewharton:disklrucache:2.0.2'
-    compile 'com.google.android.exoplayer:exoplayer:r2.2.0'
     compile "com.android.support:appcompat-v7:${supportLibraryVersion}"
-    compile 'com.getbase:floatingactionbutton:1.10.1'
     compile group: 'commons-io', name: 'commons-io', version: '2.5'
 
     /// dependencies for local unit tests
-    testCompile 'junit:junit:4.12'
-    testCompile 'org.mockito:mockito-core:1.10.19'
-
 
     /// dependencies for instrumented tests
     // JUnit4 Rules
-    androidTestCompile 'com.android.support.test:rules:0.5'
 
     // Android JUnit Runner
-    androidTestCompile 'com.android.support.test:runner:0.5'
 
     // Espresso core
-    androidTestCompile 'com.android.support.test.espresso:espresso-core:2.2.2'
 
     // Espresso web
-    androidTestCompile 'com.android.support.test.espresso:espresso-web:2.2.2'
 
     // UIAutomator - for cross-app UI tests, and to grant screen is turned on in Espresso tests
-    androidTestCompile 'com.android.support.test.uiautomator:uiautomator-v18:2.1.2'
 
     // fix conflict in dependencies; see http://g.co/androidstudio/app-test-app-conflict for details
-    androidTestCompile "com.android.support:support-annotations:${supportLibraryVersion}"
+    androidTestCompile "com.android.support:support-annotations:" +
```

Comment:
Why have you moved all these dependencies and kept the comments above? The different dependencies were ordered with different comments to know why they have been included. Can you reorder them as before?

Reply:
I have fixed this